Source/WebCore/css/makeprop.pl

```diff
@@ -427,18 +424,17 @@ sub sortByDescendingPriorityAndName
 open GPERF, ">CSSPropertyNames.gperf" || die "Could not open CSSPropertyNames.gperf for writing";
 print GPERF << "EOF";
 %{
-/* This file is automatically generated from $inputFile by makeprop, do not edit */
+// This file is automatically generated from $inputFile by the makeprop.pl script. Do not edit it.
+
 #include "config.h"
-#include \"CSSProperty.h\"
 #include \"CSSPropertyNames.h\"
-#include \"HashTools.h\"
+
+#include \"CSSProperty.h\"
```

Comment:
Any idea why only some of these quote #includes are escaped?

Reply:
None whatsoever. I didn’t even notice!